devtools/test_dashboard/buttons.js

```diff
@@ -11,7 +13,7 @@ anchor.style.width = '1000px';
 
 function plotButtons(plots, figDir) {
 
-    Object.keys(plots).forEach( function (plotname) {
+    Object.keys(plots).forEach(function (plotname) {
```

Comment:
@cldougl if you're using sublime text, you can edit your snippets to match that style using the [Package Resource Viewer](https://github.com/skuroda/PackageResourceViewer) package - I had to change it for mine.